src/Storefront/Page/Product/Configurator/ProductPageConfiguratorLoader.php

```diff
@@ -179,7 +188,30 @@ static function (PropertyGroupOptionEntity $a, PropertyGroupOptionEntity $b) use
             );
         }
 
-        return new PropertyGroupCollection($sorted);
+        $groupCollection = new PropertyGroupCollection($sorted);
+        // check if the parent product has a configuratorGroupConfig
+        $criteria = (new Criteria([$product->getParentId() ?? $product->getId()]));
+        /** @var ProductEntity $parentProduct */
+        $parentProduct = $this->productRepository
+            ->search($criteria, $context->getContext())
+            ->first();
+
+        $configuratorGroupConfig = $parentProduct->getConfiguratorGroupConfig();
```

Comment:
If you add the Inherited flag to this field, you don't have to fetch the parent at this point. 
The configuration is only saved to the parent product.

Reply:
A few tests for this change would also be important.

Reply:
Added Inherited field, added test for property group order.